cpp/src/io/text/multibyte_split.cu

```diff
@@ -55,6 +55,10 @@
 #include <numeric>
 #include <optional>
 
+namespace cudf {
+namespace io {
+namespace text {
```

Comment:
I like combining these, it tells me more about the scope of this file (specifically that we won't be putting things into `cudf::io` or `cudf`, just `cudf::io::text`.
```suggestion
namespace cudf::io::text {
```